packages/paste-core/utilities/flex/stories/index.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs, select, boolean} from '@storybook/addon-knobs';
+import {Box} from '@twilio-paste/box';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Flex, displayOptions, vAlignOptions, hAlignOptions} from '../src';
+
+const flexDisplayOptions = ['flex', 'inline-flex'];
+const flexVerticalAlignOptions = ['top', 'center', 'bottom'];
+const flexHorizontalAlignOptions = ['left', 'center', 'right'];
+
+storiesOf('Utilities|Flex', module)
+  .addDecorator(withKnobs)
+  .add('Default', () => {
```

Comment:
Please make sure to add example stories other than the knob story here.  It's a new direction we're moving in, similar to how Sendgrid has done it. Since our VRT relies on storybook stories to operate and it can't really toggle the knob values, we should have stories that provide real usage examples.  Beyond just VRT, use-case based examples here will help consumers learn and be inspired by our components.